ivy/functional/frontends/torch/linalg.py

```diff
@@ -195,6 +195,27 @@ def solve(input, other, *, out=None):
     return ivy.solve(input, other, out=out)
 
 
+
+
+@ivy.to_ivy_arrays_and_back
+@ivy.with_unsupported_dtypes({"1.11.0 and below": ("bfloat16", "float16")}, "torch")
+def solve_ex(A, B, left=True, check_errors=False, out=None):
+    result = ivy.linalg.solve(A, B)
+
+    info = ivy.zeros(result.shape[:-2], dtype=ivy.int32)
+
+    if check_errors and ivy.any(info != 0):
+        raise RuntimeError("Error occurred while solving the system.")
+
+    if out is not None:
+        assert isinstance(out, tuple) and len(out) == 2, "Invalid 'out' argument"
+        out[0] = result
+        out[1] = info
+
+    named_tuple = ivy.namedtuple("SolveExOutput", ["result", "info"])
```

Comment:
this is not a valid function in ivy, but a function from the `collections` package, feel free to import it from the package (`from collections import namedtuple`)